doc/src/vpr/command_line_usage.rst

```diff
@@ -821,6 +821,66 @@ Setting any of the following options selects `Dusty's annealing schedule <dusty_
 
     **Default:** ``0.25``
 
+.. option:: --place_cost_exp <float>
+
+    Wiring cost is divided by the average channel width over a net's bounding box
+    taken to this exponent.Only impacts devices with different channel widths in 
+    different directions or regions. 
+
+    **Default:** ``1``
+
+.. option:: --RL_agent_placement {on | off}
+
+    Uses a Reinforcement Learning (RL) agent in choosing the appropiate move type in placement.
+    It activates the RL agent placement instead of using fixed probability for each move type.
+
+    **Default:** ``on``
+
+.. option:: --place_agent_multistate {on | off}
+
+    Enable multistate agent in the placement. A second state will be activated late in
+    the annealing and in the Quench that includes all the timing driven directed moves.
+
+    **Default:** ``on``
+
+.. option:: --place_agent_algorithm {e_greedy | softmax}
+
+    Controls which placement RL agent is used. 
+
+    **Default:** ``softmax``
+
+.. option:: --place_agent_epsilon <float>
+
+    Placement RL agent's epsilon for epsilon-greedy agent. Epsilon represents
+    the percentage of exploration actions taken vs the exploitation ones.
+
+    **Default:** ``0.3``
+
+.. option:: --place_agent_gamma <float>
+
+    Controls how quickly the agent's memory decays. Values between [0., 1.] specify
+    the fraction of weight in the exponentially weighted reward average applied to moves
+    which occured greater than moves_per_temp moves ago. Values < 0 cause the
+    unweighted reward sample average to be used (all samples are weighted equally)
+
+    **Default:** ``0.05``
+
+.. option:: --place_reward_fun {basic | nonPenalizing_basic | runtime_aware | WLbiased_runtime_aware}
+
+    The reward function used by placement RL agent to learn best action at each anneal stage. 
+
+    .. note:: The latter two are only available for timing-driven placement. 
+    
+    **Default:** ``WLbiased_runtime_aware``
+
+.. option:: --place_agent_space {move_type | move_block_type}
+
+    Agent exploration space can be either based on only move types or also consider different block types.
```

Comment:
"The RL agent exploration space can either be based only on move types or can also consider the block type being moved."